content/60.methods.md

```diff
@@ -2,179 +2,264 @@
+The Python package versions are available in <https://github.com/greenelab/covid19-review/blob/external-resources/environment.yml>.<!-- To Do: These files are archived with [Zenodo?](...). -->
+
+Another issue that emerged was the need for a standardized way to cite clinical trials.
+Clinical trials that are registered with clinicaltrials.gov [@url:https://clinicaltrials.gov] receive a unique clinical trial identifier, or "NCT ID".
+Because clinical trials are registered long before results are available in manuscript form, it was important to this project to be able to refer to the clinical trial identifiers associated with a large number of relevant trials (Figure @fig:ebm-trials).
+Manubot uses the Zotero translation server [@url:https://www.zotero.org; @url:https://github.com/zotero/translation-server] to extract metadata for some types of citations.
+However, Zotero did not support clinical trial identifiers and could not extract relevant metadata from the clinical trial's URL.
+In order to enable Manubot to pull metadata associated with clinical trials based on their identifiers, we added Zotero support for these identifiers.
+Other researchers identified the same need [@url:https://forums.zotero.org/discussion/74933/import-from-clinical-trials-registry; @url:https://forums.zotero.org/discussion/77721/add-reference-from-clinical-trials-org].
+To implement this feature, we query clinicaltrials.gov to retrieve XML metadata associated with each identifier using JavaScript [@url:https://github.com/zotero/translators/pull/2153].
+Then, when Manubot requests clinical trial metadata from the Zotero translation server, it receives a more informative response that includes the trial sponsors, responsible investigators, title, and summary.
+We extended Manubot to support directly citing any of the hundreds of Compact Uniform Resource Identifier registered with <https://identifiers.org/>, including the `clinicaltrials` identifier.
+This extension enables citing a trial as `clinicaltrials:NCT04280705` instead of `https://clinicaltrials.gov/ct2/show/NCT04280705`.
+
+Another challenge that emerged was that, because of the large number of citations used in this manuscript and the fast-moving nature of COVID-19 research, keeping track of retractions, corrections, and notices of concern became a priority.
+A new plugin was added to Manubot to support "smart citations" in the HTML build of manuscripts.
+The plugin uses the [Scite](https://scite.ai/) [@doi:10.1101/2021.03.15.435418] service to display a badge below any citation with a DOI.
+The badge contains a set of icons and numbers that indicate how many times that source has been mentioned, supported, or disputed and whether there have been any important editorial notices, such as retractions or corrections.
+Using this, we were able to identify references that needed to be reevaluated.
```

Comment:
In the idea world, we'd cite a retracted paper here so that folks could see the rendering but that won't work with ACM's template. What about citing a retracted paper and then putting a link to the instance in the manubot html? Maybe this is too self referential.